tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -3998,6 +3999,26 @@ def test_url_config_lifecycle(self, create_lambda_function, snapshot, aws_client
             aws_client.lambda_.get_function_url_config(FunctionName=function_name)
         snapshot.match("failed_getter", ex.value.response)
 
+    @markers.aws.only_localstack
+    def test_create_url_config_custom_id_tag(self, create_lambda_function, aws_client):
+        custom_id_value = "my_custom_subdomain"
```

Comment:
Good catch on the ``_``, I remembered this in the docs PR but my muscle memory got the best of me in the test code.

> Do you think it'd be nice to add validation of the Custom Tag early on?

I think it'd be nice to retain the existing laziness of tag evaluation, but I can add validation to the URL config endpoint.

Reply:
I see, yes. I'm a bit conflicted about deferring the validation, because raising an exception when calling `create_function_url_config` is not something you can fix by setting different parameters for that same call, you'll need to circle back and update your tags before you create your function URL config.

I'm not particularly against it either, as long as we have validation to avoid weird issues for the user in the end, that's what matters.

Reply:
Correct, but function creation time isn't the only point in time where the user could set the tag. So unless we're capturing the value and storing it as separate function state (and probably requiring the tag to be added at definition time, in that case), the choice is either validate every time tags are modified (ie multiple endpoints), or validate when creating the function.

Reply:
I see, thanks for clarifying! This is a very different implementation than the API Gateway one, and it's fun to see the difference (the value being used once at creation for example, vs being used later in a different endpoint).

I can see it being simpler when creating the function URL config and kept in one endpoint, so that makes sense.